spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
Just curious: in Rails world, do we care about the expected relationship being changed unintentionally by someone touching other part of the code base? If we do, what is the best way to guard against that?